plane/source/docs/guide-rudder-only-plane.rst

```diff
@@ -0,0 +1,125 @@
+.. _guide-rudder-only-plane:
+
+==============================
+Rudder Only Planes (3 Channel)
+==============================
+
+A less common type of fixed wing plane is a 3 channel plane. As the
+name implies, it has 3 output channels. While pitch and throttle are directly controlled,
+roll and yaw are coupled.
+They are controlled through a single channel on the transmitter.
+Compared to using 4 channel airplanes, 3 channel planes use less servos, have less failure points,
+but provide less direct control over the vehicle's motion and less redundance against servo failure.
+
+.. warning:: Remove the propeller from your aircraft before
+             starting your setup.
+
+Setting Up Your Plane
+=====================
+
+The most important step to setting up the plane is having the correct inputs, 
+outputs, and reversals. Inputs are covered on the :ref:`RC input setup page <rc-throw-trim>`.
+
+For a 3-channel plane, configure the following parameters:
+
+* :ref:`RUDDER_ONLY<RUDDER_ONLY>` =1
+* :ref:`KFF_RDDRMIX<KFF_RDDRMIX>` =1.0
+* :ref:`YAW2SRV_DAMP<YAW2SRV_DAMP>` =0.5
+
+:ref:`YAW2SRV_DAMP<YAW2SRV_DAMP>` can be tuned later; the above value is a good starting value.
+
+If you prefer to control the throttle with one stick and the attiude of the vehicle with the other stick,
+set up your transmitter to send roll stick inputs to the :ref:`RCMAP_YAW<RCMAP_YAW>` channel (normally channel 4).
+The rudder servo should be attached to the :ref:`RCMAP_YAW<RCMAP_YAW>` channel as well (normally channel 4).
```

Comment:
rudder servo is not "attached"...the TX should send rudder stick commands to the RCMAP_YAW rc in channel

Reply:
Ok, let's fix the param description too, I agree on that word: https://ardupilot.org/plane/docs/parameters-Plane-stable-V4.5.7.html#rudder-only-rudder-only-aircraft